src/main/scala/org/broadinstitute/hail/driver/ExportPlink.scala

```diff
@@ -25,6 +58,85 @@ object ExportPlink extends Command {
 
   def requiresVDS = true
 
+  def getFamRows(vds: VariantDataset, options: Options) : IndexedSeq[String] = {
```

Comment:
I think it's a bit cleaner to put this in the main body of the `run` code, instead of passing in a `VDS` and `Options`.  I think it can also be a bit cleaner:

```
val famQuerier = Option(options.rFamID).map(code => vds.querySA(code))
...
(When you need to compute fam)
 { case (s, sa) => 
    famQuerier.map(_(sa)).getOrElse(defaultFam)
etc
```